packages/editor/src/components/block-list/block.js

```diff
@@ -412,6 +413,10 @@ export class BlockListBlock extends Component {
 					const shouldShowInsertionPoint = ( isPartOfMultiSelection && isFirstMultiSelected ) || ! isPartOfMultiSelection;
 					const canShowInBetweenInserter = ! isEmptyDefaultBlock && ! isPreviousBlockADefaultEmptyBlock;
 
+					const annotationsClassNames = annotations.map( ( annotation ) => {
```

Comment:
Ideally, this is done with a filter, I'm thinking we can add it properly as a follow up. I think the editor module shoudn't know anything about annotations.